### ISSUE TITLE
build not working

### ISSUE THREAD

**Issue by amitsjedhe** [gjovanov/facer#10]
with error :
warning " > vuetify@1.5.21" has unmet peer dependency "vue@^2.5.18".
error An unexpected error occurred: "EPERM: operation not permitted, open 'D:\\AI\\facer-master\\facer-master\\node_modules\\acorn-jsx\\node_modules\\.bin\\acorn.cmd'".

**Comment by gjovanov** [comment]
Can you pls be more specific on what steps did u perform and at what point did experience this error?

Did you install npm packages with: `npm i`?

**Comment by amitsjedhe** [comment]
Hi Goran Jovanov,

Your demo application is awesome, i tried to run same on my machine with
the repo which you have shared.
but unfortunately it's breaking, please find the attachment will logs
available during execution, please help to get this rectified and if i'll
be able to run this then it will be great achievement for me.

Thanks & Regards
Amit.S.Jedhe.

**Comment by gjovanov** [comment]
Hi @amitsjedhe,
you didn't provide any logs yet.

To assist u with it, I would need more details.

**Comment by amitsjedhe** [comment]
I have already share you logs in previous mail, same are attached in this
email as well.
2019-11-19T08_32_55_370Z-debug
error.log

Thanks & Regards
Amit.S.Jedhe.

**Comment by gjovanov** [comment]
Hi @amitsjedhe,
I assume u r using Email Reply to the Email notifications received from Github? If so, then for some reason Github is only sending me your text message without any attachment.

Pls use the the issue on Github you opened and attach the info there.

Thanks.

**Comment by amitsjedhe** [comment]
Ohh, will send you tracking through GitHub, or u can share u r email id
where i can forward it.

**Comment by gjovanov** [comment]
Let's use Github. Others might also benefit if they encounter the same issue.

**Comment by amitsjedhe** [comment]
[error.log](https://github.com/gjovanov/facer/files/3873050/error.log)
[2019-11-19T08_32_55_370Z-debug.log](https://github.com/gjovanov/facer/files/3873051/2019-11-19T08_32_55_370Z-debug.log)

**Comment by gjovanov** [comment]
Hi @amitsjedhe,
I have looked at the logs and it seems that u have encountered an issue during `npm install` of the package `sharp`, for which version there was no available compiled version for download, so **npm** had to compile it for you via node-gyp. And for some reason that compilation process failed.


Anyways, since most of the packages in `package.json` were outdated, I've spent some time to upgrade them to their latest version, which also required some refactoring in the code.

Long story short, if you `git pull` the latest changes, deleted your `node_modules` foder and try again to install all dependencies `npm i`, after that you should be able to start the app via:
1. `npm run api`
2. `npm run dev`

Pls let me know if this works for you now or not.

**Comment by amitsjedhe** [comment]
yes it's working now, but accuracy of face recolonization is bit low on  my machine, most of the time shows unknown, might be due to camera quality

**Comment by amitsjedhe** [comment]
it through below error but continue after some time
PayloadTooLargeError: request entity too large
    at readStream (D:\AI\facer-master1\facer-master\node_modules\raw-body\index.js:155:17)
    at getRawBody (D:\AI\facer-master1\facer-master\node_modules\raw-body\index.js:108:12)
    at read (D:\AI\facer-master1\facer-master\node_modules\body-parser\lib\read.js:77:3)
    at jsonParser (D:\AI\facer-master1\facer-master\node_modules\body-parser\lib\types\json.js:135:5)
    at Layer.handle [as handle_request] (D:\AI\facer-master1\facer-master\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\AI\facer-master1\facer-master\node_modules\express\lib\router\index.js:317:13)
    at D:\AI\facer-master1\facer-master\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\AI\facer-master1\facer-master\node_modules\express\lib\router\index.js:335:12)
    at next (D:\AI\facer-master1\facer-master\node_modules\express\lib\router\index.js:275:10)
    at expressInit (D:\AI\facer-master1\facer-master\node_modules\express\lib\middleware\init.js:40:5)
D:\AI\facer-master1\facer-master\data\faces.json
PayloadTooLargeError: request entity too large
    at readStream (D:\AI\facer-master1\facer-master\node_modules\raw-body\index.js:155:17)
    at getRawBody (D:\AI\facer-master1\facer-master\node_modules\raw-body\index.js:108:12)
    at read (D:\AI\facer-master1\facer-master\node_modules\body-parser\lib\read.js:77:3)
    at jsonParser (D:\AI\facer-master1\facer-master\node_modules\body-parser\lib\types\json.js:135:5)
    at Layer.handle [as handle_request] (D:\AI\facer-master1\facer-master\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\AI\facer-master1\facer-master\node_modules\express\lib\router\index.js:317:13)
    at D:\AI\facer-master1\facer-master\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\AI\facer-master1\facer-master\node_modules\express\lib\router\index.js:335:12)
    at next (D:\AI\facer-master1\facer-master\node_modules\express\lib\router\index.js:275:10)
    at expressInit (D:\AI\facer-master1\facer-master\node_modules\express\lib\middleware\init.js:40:5)
D:\AI\facer-master1\facer-master\data\faces.json
D:\AI\facer-master1\facer-master\data\faces.json
![Capture](https://user-images.githubusercontent.com/47139678/69418215-3f57ad00-0d40-11ea-8bf2-f36e6b6cb037.JPG)
